Gemfile

```diff
@@ -17,7 +17,9 @@ group :development, :test do
   gem 'coveralls', require: false
   gem 'pry-rails'
   gem 'pry-byebug'
+  gem 'rspec'
   gem 'rspec-rails'
+  gem 'rspec-rails-matchers'
```

Comment:
What does this gem do for you? Why is it included?

Reply:
The rspec gem was already there before, I thought something was missing. I don't understand which gem is missing